modules/3d-tiles/src/lib/ion/ion.ts

```diff
@@ -23,12 +23,13 @@ export async function getIonTilesetMetadata(accessToken, assetId) {
 
   // Step 2: Query metdatadata for this asset.
   const ionAssetMetadata = await getIonAssetMetadata(accessToken, assetId);
-  const {type, url} = ionAssetMetadata;
+  const type = ionAssetMetadata.type;
+  const url = ionAssetMetadata.options?.url;
```

Comment:
Does it make sense to be backwards compatible
```suggestion
  // As of Oct 2024 ion service now returns the resource URL in an options object
  const url = ionAssetMetadata.options?.url || ionAssetMetadata.url;
```